src/modules/hip/handlehip.cpp

```diff
@@ -240,6 +240,7 @@ struct HandleImpl
 
         hipMalloc(&(this->initHandle->mem.mgpu.rgbArr.rgbmem), sizeof(RpptRGB) * this->nBatchSize);
         hipMalloc(&(this->initHandle->mem.mgpu.scratchBufferHip.floatmem), sizeof(Rpp32f) * 8294400);    // 3840 x 2160
+        hipHostMalloc(&(this->initHandle->mem.mgpu.scratchBuf.uintmem), sizeof(Rpp32u) * 8294400);    // 3840 x 2160
```

Comment:
Pls call this scratchBufPinned, and change to floatmem like above
Reinterpret cast while using it.